app/View/Components/Forms/EnvVarInput.php

```
@@ -26,6 +26,7 @@ public function __construct(
         public bool $disabled = false,
         public bool $readonly = false,
         public ?string $helper = null,
+        public bool $allowToPeak = true,
```

PHPMD (2.15.0)

The method __construct has a boolean flag argument $allowToPeak, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)